synapse/app/admin_cmd.py

```diff
@@ -1,4 +1,3 @@
-#!/usr/bin/env python
```

Comment:
looks like you're setting the `+x` bit on this? That seems to be in opposition to the removal of the shebang?

Reply:
Good spot, that wasn’t my intention. Fixed it.